lib/components/EmailInput.js

```diff
@@ -21,16 +21,31 @@ import { hyphenize } from "../common";
 import Typography from "./Typography";
 import Label from "./Label";
 
-const CustomControl = ({ children, ...props }) => (
-  <components.Control {...props}>
-    <div className="neeto-ui-overflow-hidden">{children}</div>
-  </components.Control>
-);
+const CustomControl = ({ children, ...props }) => {
+  const { prefix } = props.selectProps;
+
+  return (
+    <components.Control {...props}>
+      {prefix && <div className="pl-2">{prefix}</div>}
```

Comment:
Since we have removed tailwindcss from neetoUI, please remove the `pl-2` and update it with a custom class